darwin/dataset/remote_dataset.py

```diff
@@ -205,17 +197,23 @@ def pull(
 
     def remove_remote(self):
         """Archives (soft-deletion) the remote dataset"""
-        self.client.put(f"projects/{self.project_id}/archive", payload={})
+        self.client.put(f"datasets/{self.dataset_id}/archive", payload={})
+
+    def get_report(self, granularity="day"):
```

Comment:
could benefit from a docstring as the purpose of this method is not trivial (at least to me)